smt/examples/airfoil_parameters/learning_airfoils_parameters.py

```diff
@@ -0,0 +1,249 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from smt.surrogate_models.genn import GENN, load_smt_data
+import csv
+from sklearn.model_selection import train_test_split 
+import pickle
+import tensorflow as tf
+import matplotlib.pyplot as plt
+import pandas as pd
+
+"""
+Predicting Airfoil Aerodynamics through data
+by Raul Carreira Rufato and Prof. Joseph Morlier
+
+This notebook is a tutorial to determine the aerodynamic coefficients of a given airfoil using SMT. The obtained surrogate model can be used to give predictions for certain Mach numbers, angles of attack and the aerodynamic coefficients. These calculations can be really usefull in case of an airfoil shape optimization. The input parameters uses the airfoil Camber and Thickness mode shapes.
+
+Inputs: Airfoil Camber and Thickness mode shapes, Mach, alpha
+
+Outputs (options): cd, cl, cm
+
+In this test case, we will be predicting only the Cd coefficient. However, the other databases for the prediction of the other terms are available in the same repository.
+
+Bouhlels mSANN in a nutshell:
+
+a. Using inverse distance weighting (IDW) to interpolate the surface function of each airfoil.
```

Comment:
below `GENN` is used, not `IDW`? Maybe add a short explicit description of what is `mSANN` exactly, there are 4 steps below but do not understand how they relate to it. mSANN, what is it and what is it for?